### PR TITLE
[ETP-474] Tests/v2 snapshots p4

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ETP-474]

[ETP-474]: https://moderntribe.atlassian.net/browse/ETP-474
Screen Shot 2020-10-21 at 1 32 57 AM

### CODE REVIEW

File: src/views/v2/tickets/submit/button.php [hunk 1]
```diff
@@ -8,48 +8,47 @@
...
- * @param array $button_name The button classes.
  */
 $button_classes = apply_filters(
-	'tribe_tickets_ticket_block_submit_classes',
-	[
-		'tribe-common-c-btn',
-		'tribe-common-c-btn--small',
-		'tribe-tickets__buy',
-	]
+		'tribe_tickets_ticket_block_submit_classes',
```
Comment (tr1b0t) [inline src/views/v2/tickets/submit/button.php:38]: :no_entry_sign: Multi-line function call not indented correctly; expected 4 spaces but found 8

<sub>hash: 19dbff87af038fd60468c9e661a64d47</sub>

File: src/views/v2/tickets/submit/button.php [hunk 2]
```diff
@@ -8,48 +8,47 @@
...
+		[
```
Comment (tr1b0t) [inline src/views/v2/tickets/submit/button.php:39]: :no_entry_sign: Multi-line function call not indented correctly; expected 4 spaces but found 8

<sub>hash: 78c6f6a5258d68069d6a9c80d84e0b59</sub>

File: src/views/v2/tickets/submit/button.php [hunk 3]
```diff
@@ -8,48 +8,47 @@
...
+				'tribe-common-c-btn',
```
Comment (tr1b0t) [inline src/views/v2/tickets/submit/button.php:40]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: 476ef407dd4109b21b1c8f9af613a3cc</sub>

File: src/views/v2/tickets/submit/button.php [hunk 4]
```diff
@@ -8,48 +8,47 @@
...
+				'tribe-common-c-btn--small',
```
Comment (tr1b0t) [inline src/views/v2/tickets/submit/button.php:41]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: 2fde57e0360041fde7058db70c58bc48</sub>

File: src/views/v2/tickets/submit/button.php [hunk 5]
```diff
@@ -8,48 +8,47 @@
...
+				'tribe-tickets__buy',
```
Comment (tr1b0t) [inline src/views/v2/tickets/submit/button.php:42]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: cf0b8c59aa415999da8f934f8349ee05</sub>

File: src/views/v2/tickets/submit/button.php [hunk 6]
```diff
@@ -8,48 +8,47 @@
...
+		]
 );
 ?>
 <button
-	<?php tribe_classes( $button_classes ) ?>
-	id="tribe-tickets__buy"
-	type="submit"
-	name="<?php echo esc_html( $submit_button_name ); ?>"
-	<?php tribe_disabled( true ); ?>
+		<?php tribe_classes( $button_classes ) ?>
```
Comment (tr1b0t) [inline src/views/v2/tickets/submit/button.php:47]: :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 72a456ff980f0b4828673e6d2b8bddb6</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 6 error(s)

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**